core/cloudflow-flink-tests/src/test/java/cloudflow/flink/javadsl/FlinkProcessor.java

```diff
@@ -45,15 +45,16 @@
   // Step 3: Provide custom implementation of `FlinkStreamletLogic` that defines
   //         the behavior of the streamlet
   @Override public FlinkStreamletLogic createLogic() {
-    return new FlinkStreamletLogic(getStreamletContext()) {
+    final FlinkStreamletContext context =  getStreamletContext();
```

Comment:
This also does not seem necessary, if I revert this change, everything compiles?

Reply:
done